cartography/intel/gsuite/api.py

```diff
@@ -0,0 +1,243 @@
+import logging
+
+from googleapiclient.discovery import HttpError
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_all_groups(admin):
+    """
+    Return list of Google Groups in your organization
+    Returns empty list if we are unable to enumerate the groups for any reasons
+
+    :param admin: apiclient discovery resource object
+    See
+    :return: List of Google groups in domain
+
+    """
+    request = admin.groups().list(customer='my_customer', maxResults=20, orderBy='email')
```

Comment:
`my_customer` means the current GSuite account that the key is tied to.  
https://developers.google.com/admin-sdk/directory/v1/guides/manage-customers

Otherwise you need to populate it with the `uid` of your gsuite.